rest/route/middleware.go

```diff
@@ -394,7 +394,10 @@ func urlVarsToProjectScopes(r *http.Request) (string, int, error) {
 	}
 
 	projectID := util.CoalesceStrings(append(query["project_id"], query["projectId"]...), vars["project_id"], vars["projectId"])
-
+	destProjectID := util.CoalesceString(query["dest_project"]...)
+	if destProjectID != "" {
```

Comment:
@john-m-liu Could you take a look at this? I don't understand ACL middleware well enough to know if this is the right thing.

Reply:
This actually isn't the thing I'm going to push; I'll add the actual commit but it's going to break compile until the gimlet change is resolved.

Reply:
(Updated)